### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by santi81**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/santi81/TestReviewNinja/pull/4" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -5,3 +5,7 @@ Testing Review Code Ninja..so far so good
 Testing now a differrent way to review changes...via code reivew ninja
 
 Ninja Code Review Testing
+
+
+
+I am making some random changes...
```
Comment (santi81) [inline README.md]: this is crap ...just make some decent changes !fix

Comment (santi81) [inline README.md]: need more changes !resolve

**Comment by santi81** [comment]
reivrw again

File: README.md (hunk 1 above)
Comment (santi81) [inline README.md]: !resolved

Comment (santi81) [inline README.md]: !completed

**Pull request merged**